src/sentry/tasks/post_process.py

```diff
@@ -437,6 +437,29 @@ def post_process_group(
                 plugin_post_process_group(
                     plugin_slug=plugin.slug, event=event, is_new=is_new, is_regresion=is_regression
                 )
+            if features.has(
+                "organizations:sentry-functions",
+                event.project.organization,
+            ):
+
+                from sentry.models import SentryFunction
+
+                for fn in SentryFunction.objects.filter(organization=event.organization).all():
```

Comment:
You should consider spawning celery tasks to trigger these functions. post_process is very latency sensitive and doing sync calls to a remote API will eventually cause an outage.